shared/entities/project-key-costs.view.ts

```diff
@@ -0,0 +1,117 @@
+import { ValueTransformer, ViewColumn, ViewEntity } from "typeorm";
+
+export const decimalTransformer: ValueTransformer = {
+  to: (value: number | null) => value,
+  from: (value: string | null): number | null =>
+    value !== null ? parseFloat(value) : null,
+};
+
+@ViewEntity({
+  name: "project_scorecard_key_costs",
+  expression: `
+SELECT
+  p.project_name AS project_name,
```

Comment:
I think we are missing (at least) 2 columns here:

1. ecosystem
2. country_code

Without those, the api consumer won't be able to filter by this properties, which is required